### PR TITLE
Updated colorbars and full vertically integrated dusts for dust plots

### PR DESCRIPTION
**Pull request by kayeekayee**
1) Changed the range of the colorbars.
near_sfc_fine_dust
near_sfc_coarse_dust
2) Combined the vertically integrated dusts (fine and coarse) into one plot.
full_int_dust

### CODE REVIEW

**Review by cshartsough — APPROVED** [review]
These changes look good.  If you think you'll ever want plots of integrated coarse dust only, you could leave the parameters in the default_specs file (they are still there for integrated fine dust).

**Review by cshartsough — APPROVED** [review]
These changes look good.  If you think you'll ever want plots of integrated coarse dust only, you could leave the parameters in the default_specs file (they are still there for integrated fine dust).

**Review by Brian-Jamison — APPROVED** [review]
Thanks, Ka Yee!  This looks great!

**Comment by cshartsough** [comment]
It looks like the automated tests failed due to the new summed field having "_" in the name, which our pygraf interprets in some cases to look for special hours, etc.  If you change full_int_dust to something like fullintdust, etc., it should work.  Sorry I didn't think of that when I gave you the example.

File: adb_graphics/default_specs.yml [hunk 1]
```diff
@@ -480,13 +471,38 @@ dewp: # Dew point temperature
     wind: 10m
 finedust:
   int:
-    <<: *coldust
+    clevs: [1, 4, 7, 11, 15, 20, 25, 30, 40, 50, 75, 150, 250, 500]
```
Comment (christinaholtNOAA) [inline adb_graphics/default_specs.yml:474]: You could add a PyYAML anchor here and reference it later in full_int_dust to reduce duplication, like was done for coldust before this change.

**Review by christinaholtNOAA — COMMENTED** [review]
@kayeekayee This looks good to me. Just one minor suggestion below.

File: adb_graphics/default_specs.yml (hunk 1 above)
Comment (kayeekayee) [reply adb_graphics/default_specs.yml]: Thank you Christina!

**Comment by kayeekayee** [comment]
Thank you so much for reviewing this PR. Here is some updated plots for the new fire yesterday.
Vertical_intergated_dusts
fine_dust
coarse_dust

**Pull request merged**